tests/vmi_gpu_test.go

```diff
@@ -79,7 +80,15 @@ var _ = Describe("[Serial][sig-compute]GPU", Serial, func() {
 			vmi, apiErr := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(randomVMI)
 			Expect(apiErr).ToNot(HaveOccurred())
 
-			pod := libvmi.GetPodByVirtualMachineInstance(vmi, util.NamespaceTestDefault)
+			By("Waiting for VMI to be scheduled")
+			Eventually(func() bool {
```

Comment:
--> 
```
Eventually(libvmi.ThisVMI(vmi)).Should(matcher.BeInPhase(v1.Scheduling))
```

Reply:
Oh thanks for letting me know about this!
Done